xbmc/SortFileItem.h

```diff
@@ -40,6 +40,8 @@ typedef enum {
   SORT_METHOD_YEAR,
   SORT_METHOD_VIDEO_RATING,
   SORT_METHOD_DATEADDED,
+  SORT_METHOD_ALBUM_DATEADDED,
+  SORT_METHOD_ARTIST_DATEADDED,
```

Comment:
perhaps we should update the python docs and recommend to use a string instead of an int?
95% of our plugins already use that.

http://xbmclogs.com/ptjsjxwyn/vlcvps/raw